### PR TITLE
Add type definition for express-formidable

### PR DESCRIPTION
**Pull request by tdolsen**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Run `tsc` without errors.
- [x] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

### CODE REVIEW

File: express-formidable/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Type definitions for express-formidable
+// Project: https://github.com/noraesae/express-formidable
+// Definitions by: Torkild Dyvik Olsen <https://github.com/tdolsen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "express-formidable" {
```
Comment (andy-ms) [inline express-formidable/index.d.ts]: Remove this wrapper -- types should still work.

Comment (tdolsen) [inline express-formidable/index.d.ts]: Done - and they do. :+1:

File: express-formidable/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Type definitions for express-formidable
```
Comment (andy-ms) [inline express-formidable/index.d.ts]: Please specify version: `// Type definitions for express-formidable 1.0`

**Review by ghost — COMMENTED** [review]
Sorry, one thing I missed last time

File: express-formidable/index.d.ts (hunk 2 above)
Comment (tdolsen) [inline express-formidable/index.d.ts]: My bad. Done now.

**Pull request merged**